customTests/custom-test-setup.sh

```diff
@@ -24,120 +27,72 @@ function install_perf_test(){
 	VERSION=4.5-0.12
```

Comment:
should we ever update this version?

Reply:
I was thinking about this. Let me bundle this in another PR because it may require more testing.